### DISCUSSION TITLE
Only allow certain fields to be requested..

### DISCUSSION THREAD

**Discussion by irishburlybear** [pocketbase/pocketbase#1605] [Q&A]
I'm looking for a way to only get the fields I need from a request.
I thought there would be like a:  pb.collection('table').getOne(id, {fields:'a_field_i_only_want,maybe_a_second_field'})

**Comment by ganigeorgiev** [comment]
Unfortunately this at the moment is not supported (_you can also check https://github.com/pocketbase/pocketbase/discussions/354_) but it is in my todo and eventually will be added after the planned "view" collection type in the near future.

**Reply by irishburlybear** [reply]
No worries, I'm using sveltekit and just for the work-around: I re-map my results and send that down the pipe..
